flask-app/src/customers/customers.py

```diff
@@ -37,6 +37,59 @@ def signup():
 
     return f'Successfully registered customer {first} {last}!'
 
+# get a profile
+@customers.route('/<customerID>', methods=['GET'])
+def get_profile(customerID):
```

Comment:
should these get customers routes (/c/<customerID, /m/customers) return credit card information?

Reply:
Since it is a manger route I would say yes, but it is up to you.